### ISSUE TITLE
Host name edit does not save the new value

### ISSUE THREAD

**Issue by cdeadmin** [openbmc/openbmc#3104]
<cde:info> SW425052 </cde:info>

**Comment by gtmills** [comment]
From the overview page
Click on the edit link next to the Server name (called Server name but it is actually the hostname from  `/xyz/openbmc_project/network/config/attr/HostName`)
![overview_page_hostname](https://user-images.githubusercontent.com/5455127/38893091-e905ce04-424e-11e8-9c21-dd9af04f29a8.PNG)

![edit_server_name](https://user-images.githubusercontent.com/5455127/38893234-50f9b598-424f-11e8-83b7-1320135fdc4d.PNG)

When you enter a new name and click, nothing happens. 
```
curl -c cjar -b cjar -k -H "Content-Type: application/json" -X  GET https://$bmc/xyz/openbmc_project/network/config/attr/HostName
{
  "data": "witherspoon"

```

**Comment by gtmills** [comment]
Commented this edit button out for now, https://gerrit.openbmc-project.xyz/#/c/10197/1

**Comment by gtmills** [comment]
same issue here https://github.com/openbmc/openbmc/issues/2730

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/10442> Allow user to modify hostname in overview page
Resolves  #3104 Host name edit does not save the new value
